doc/src/modules/physics/mechanics/sympy_mechanics_for_autolev_users.rst

```diff
@@ -309,8 +309,8 @@ Mathematical Equivalents
 | solve for.            |                       | refer to              |
 |                       |                       | ``nonlinsolve`` and   |
 |                       |                       | ``nsolve`` in         |
-|                       |                       | `solvers. <http://docs|
-|                       |                       | .sympy.org/latest/    |
+|                       |                       | `solvers. <https://   |
+|                       |                       | docs.sympy.org/latest/|
```

Comment:
I guess it's referring to this. Sphinx doesn't like it when you have two links to the same URL with different names for some reason. 

Actually, this link is rather silly. It should be a relative link, not a link to docs.sympy.org. It looks like this needs to be fixed throughout this document (and possibly elsewhere in the docs as well).

Reply:
Opened https://github.com/sympy/sympy/issues/15380 for the docs.sympy.org links. If you don't want to fix the other links for this PR that's fine, just this one link needs to be made relative to fix the build.

Reply:
ok, just to clarify this should link to `/latest/modules/solvers/solvers.html` or is there an `rst` way to link to the specific document?

Reply:
There's a way to cross reference the document directly. You might have to add a label to the top of solvers.rst.